### DISCUSSION TITLE
Image caption seems to break page

### DISCUSSION THREAD

**Discussion by brandonpow1** [retypeapp/retype#245] [Q&A] [answered]
i like retype! it's nice. but i got a problem:

[website](https://brandonpow1.github.io/vn-guide/technical/presplash/), [presplash.md](https://github.com/brandonpow1/vn-guide/blob/main/technical/presplash.md) line 19

`![](</assets/deutschmenue.webp>)`
gives me:

![image](https://user-images.githubusercontent.com/76508025/147364000-420d09c4-7bc7-4467-bea5-aee0a555e553.png)

if i try the same code with an alt text, the whole page appears blank.
[website](https://brandonpow1.github.io/vn-guide/technical/presplash-broken/), [presplash-broken.md](https://github.com/brandonpow1/vn-guide/blob/main/technical/presplash-broken.md) line 19

`![What do I click?!](</assets/deutschmenue.webp>)`
gives me:

![image](https://user-images.githubusercontent.com/76508025/147364262-dbfed406-5459-4085-a4d1-214af5d5d755.png)

the entire screen flashes with the content for a split second, then appears blank. i'm using ms edge. can i get some help please? thank you!

**Comment by geoffreymcgill** [comment]
Hi @brandonpow1. Thanks for the report. Can you try the same configurations but without the `<` and `>` characters? Those characters should not be required.

### Existing

```
![](</assets/deutschmenue.webp>)
![What do I click?!](</assets/deutschmenue.webp>)
```

### Revised

```
![](/assets/deutschmenue.webp)
![What do I click?!](/assets/deutschmenue.webp)
```

Is there a reason why you would require the `<` and `>` to be in the path?

**Reply by brandonpow1** [reply]
oh, er... i just kinda migrated from gitbook and those were there in the .md files from the start. i shouldn't add those less-than or more-than signs next time.

**Reply by geoffreymcgill** [reply]
> i shouldn't add those less-than or more-than signs next time.

Do you remember adding the `<` and `>` characters? Or, were they somehow added by GitBook?

If they were added by GitBook for some reason, we could add a simple process to strip them out of the final generated HTML. If you added them, then it's probably best if Retype does not modify.

**Reply by brandonpow1** [reply]
ah my bad wording it like that. the < and > were added by gitbook. i used the gitbook online editor. when i synced it to github and downloaded the .md files they were there already.

**Reply by DannyBen** [reply]
The `<...>` notation in markdown is a used for [creating a link out of URLs](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses) - perhaps the online editor did not remove them when they were converted from URL only to the `[title](url)` notation.

I don't think retype should do anything in this regard.

**Reply by geoffreymcgill** [reply]
@DannyBen I did not know about the `<...>` link notation. I ran some tests and Retype already correctly handles that syntax option, as well as `![](<...>)` image format. So, nothing needs to be done and the format is already supported by Retype. 

The original error was caused by the `"` in the caption value, so we still need to figure out why that was failing and fix.

I will keep this thread updated with our progress with the caption fix.

**Comment by geoffreymcgill** [comment, marked as answer]
I took a closer look at your `.md` page and I was able to isolate the error to a sample when a `"` character is used within the caption. 

One of your image component configurations with a `"` in the caption was causing the page to fail. The failing sample can be simplified to the following:

```
// FAIL
!["test")](/assets/inhaltswarnung.webp)

// PASS
![test)](/assets/inhaltswarnung.webp)
```

As a temporary work-around, removing the `"` characters from the image component caption should get your page working.

We will investigate why the `"` chars are not being escaped in this scenario and will try our best to fix as quickly as possible.

Thanks again for reporting the issue and providing a sample demonstrating how to reproduce.

**Reply by brandonpow1** [reply]
support on christmas day? wow you guys must be very committed to this project hehe

your temporary workaround worked! i'll have to avoid using double quote marks for the time being. thank you!

**Reply by geoffreymcgill** [reply]
This double-quote issue has been fixed and the fix will be included in the next release of Retype. The current release is v1.11.2 . 

At the moment, we do not have a scheduled date for the next release, although we are working hard to get it wrapped up as soon as we can.

@brandonpow1 Thanks again for reporting the defect.
